multiqc/modules/sincei/scFilterStats.py

```diff
@@ -0,0 +1,197 @@
+            }
+            header["pct_Excluded_Motif"] = {
+                "title": "% Motif Filtered",
+                "description": "Percent of alignments lacking the expected sequence motif",
+                "scale": "PuRd",
+                "min": 0,
+                "max": 100,
+            }
+            header["pct_Excluded_GC"] = {
+                "title": "% GC Filtered",
+                "description": "Percent of alignments lacking the expected GC content",
+                "scale": "OrRd",
+                "min": 0,
+                "max": 100,
+            }
+            header["pct_Low_Aligned_Fraction"] = {
+                "title": "% Low_Aligned_Fraction",
+                "description": "Percent of alignments where the number of bases that match the reference were lower then desired",
+                "scale": "PuRd",
+                "min": 0,
+                "max": 100,
+            }
+            tdata = dict()
+            for k, v in self.sincei_scFilterStats.items():
+                tdata[k] = {
+                    "N Entries": v["total"],
+                    "pct_Filtered": v["filtered"],
+                    "pct_Blacklisted": v["blacklisted"],
+                    "pct_Below_MAPQ": v["mapq"],
+                    "pct_Missing_Flags": v["required_flags"],
+                    "pct_Forbidden_Flags": v["excluded_flags"],
+                    "pct_sincei_Dupes": v["internal_dupes"],
+                    "pct_Duplication": v["external_dupes"],
+                    "pct_Singletons": v["singletons"],
+                    "pct_Excluded_Strand": v["wrong_strand"],
+                    "pct_Excluded_Motif": v["wrong_motif"],
+                    "pct_Excluded_GC": v["unwanted_gc"],
+                    "pct_Low_Aligned_Fraction": v["low_alignedfrac"],
+                }
+            config = {"namespace": "sincei scFilterStats", "max_table_rows": 10000}
+            self.add_section(
+                name="Filtering metrics",
+                anchor="scFilterStats",
+                description="Estimated percentages of alignments filtered independently for each setting in `scFilterStats`",
+                plot=beeswarm.plot(tdata, header, config),
+            )
+
+        return len(self.sincei_scFilterStats)
+
+    def parsescFilterStatsFile(self, f):
+        d = {}
+        firstLine = True
+        for line in f["f"].splitlines():
+            if firstLine:
+                firstLine = False
+                continue
+            cols = line.strip().split("\t")
+
+            if len(cols) != 14:
+                # This is not really the output from scFilterStats!
+                log.warning(
+                    "{} was initially flagged as the tabular output from scFilterStats, but that seems to not be the case. Skipping...".format(
+                        f["fn"]
+                    )
+                )
+                return dict()
+
+            s_name = self.clean_s_name(cols[0], f)
```

Comment:
Since we have the header, perhaps it would be more robust to use the header to index values - `cols['Cell_ID']` instead of `cols[0]`, and so on. Would be more robust in case if new columns are inserted into future versions